docs/migration/migratingfrommlapi.md

```diff
@@ -0,0 +1,111 @@
+---
+id: migratingfrommlapi
+title: Updating to the Unity Package
+sidebar_label: Updating to the Unity Package
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::important
+This upgrade guide targets project using the MLAPI version which was installed by the MLAPI installer or by cloning the source code from Github.
+If your project uses a Package Manager version of MLAPI then this guide is not relevant.
+:::
+
+The aqcuisiton of MLAPI has been a unique event in Unity history. In our efforts to integrate MLAPI into the Unity eco system we are turning it in a Unity package.
+
+What this means for you is:
+- MLAPI will be easily accessible from the Unity Editor in the Package Manager window.
+- Updating MLAPI in the future will work directly from the package manager.
+- You will have full access to the MLAPI source in your project making it easier to debug and inspect MLAPI.
+- Upgrading from the last MLAPI version to the new package version is a bit complicated. We are sorry for that.
+
+# First, Backup your Project!
+:::important
+This step is **not** optional. Upgrading to the package version of MLAPI can screw up your project. Backup your project by either creating a copy of your entire project folder or by using source control software like Git. Best do both to be safe.
+:::
+
+
+# Upgrading
+
+Manually upgrading from the dll version installed by MLAPI to the new package version breaks all MLAPI component references in your scenes and prefabs. This is the case because MLAPI references components via GUIDs and those GUIDs work differently for packages then for DLLs. To allow you to upgrade smoothly to the new version of MLAPI we have created an upgrade tool.
+
+To start upgrading add the upgrade tool to your project by using the `Add package from git URL..` option in the package manager window and using the following url: https://github.com/Unity-Technologies/mlapi-community-contributions.git?path=/com.unity.multiplayer.mlapi-patcher
+
+After installing the patcher package you are good to go. Follow the following steps to upgrade.
+
+## 1.**Install the MLAPI Package**
+Follow the [installation guide](TODO how can I easily link to something telling the user how to install MLApI?) for installing the package version of MLAPI.
```

Comment:
Hmm maybe this doc page? for using the Package Manager? 
https://docs.unity3d.com/Packages/com.unity.package-manager-ui@2.1/manual/index.html

```suggestion
Locate and install the MLAPI package using the Package Manager. See the [Unity Package Manager](https://docs.unity3d.com/Packages/com.unity.package-manager-ui@2.1/manual/index.html) guide for more information.
```

Reply:
I don't think conventional docs will work here. Because MLAPI won't be visible in the package manager afaik. People will have to add it via git url. There is a section in the hello world tutorial which covers this. I wonder whether we maybe should copy that into a second plain installation page.

Reply:
Yeah that makes more sense! We can definitely copy paste from hello world here!